newsroom/news_api/api_tokens/__init__.py

```diff
@@ -23,6 +23,7 @@
 from . import views  # noqa
 
 
+# TODO-ASYNC: Remove altogether once all the modules from news_api are async
 class CompanyTokenAuth(TokenAuth):
```

Comment:
We should have a separate ticket to double check if this is still needed. It probably isn't but better to check if carefully and removed it :)